x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_by_type_panel/helpers.tsx

```diff
@@ -62,3 +62,9 @@ const getAggregateAlerts = (
 export const isAlertsTypeData = (data: SummaryChartsData[]): data is AlertsTypeData[] => {
   return data?.every((x) => has(x, 'type'));
 };
+
+export const isAlertsByTypeAgg = (
```

Comment:
`nit`: consider renaming it to `get...` because it returns the boolean and not itself a boolean.